src/constants/potentialMarkets.ts

```diff
@@ -0,0 +1,85 @@
+export type ExchangeConfigParsedCsv = Array<{
+  base_asset: string;
+  exchange: string;
+  pair: string;
+
+  adjust_by_market: string;
+  min_2_depth: string;
+  avg_30d_vol: string;
+  reference_price: string;
+  risk_assessment: string;
+  num_oracles: string;
+  liquidity_tier: string;
+  asset_name: string;
+}>;
+
+export type ExchangeConfigItem = {
+  exchangeName: string;
+  ticker: string;
+  adjustByMarket?: string;
+};
+
+export type PotentialMarketParsedCsv = Array<{
+  base_asset: string;
+  reference_price: string;
+  num_oracles: string;
+  liquidity_tier: string;
+  asset_name: string;
+  p: string;
+  atomic_resolution: string;
+  min_exchanges: string;
+  min_price_change_ppm: string;
+  price_exponent: string;
+  step_base_quantum: string;
+  ticksize_exponent: string;
+  subticks_per_tick: string;
+  min_order_size: string;
+  quantum_conversion_exponent: string;
+}>;
+
+export type PotentialMarketItem = {
+  baseAsset: string;
+  referencePrice: string;
+  numOracles: number;
+  liquidityTier: number;
+  assetName: string;
+  p: number;
+  atomicResolution: number;
+  minExchanges: number;
+  minPriceChangePpm: number;
+  priceExponent: number;
+  stepBaseQuantum: number;
+  ticksizeExponent: number;
+  subticksPerTick: number;
+  minOrderSize: number;
+  quantumConversionExponent: number;
+};
+
+export const NUM_ORACLES_TO_QUALIFY_AS_SAFE = 6;
+
+export const LIQUIDITY_TIERS = {
+  0: {
+    label: 'Large-cap',
```

Comment:
do these labels need to be internationalized?

Reply:
Yeah, will add in a follow up!